### PR TITLE
Change release server token path

### PR DESCRIPTION
**Pull request by yengliong93**
### PULL DESCRIPTION

This PR changes the release server token path from /etc/intel_edge_node/tokens/platform-update-agent/rs_access_token to /etc/intel_edge_node/tokens/release-service/access_token.

### Impact Analysis

| Info | Please fill out this column |
| ------ | ----------- |
| Root Cause | Specifically for bugs, empty in case of no variants |
| Jira ticket | Add the name to the Jira ticket eg: "NEXMANAGE-622". Automation will do the linking to Jira |

### CODE MAINTAINABILITY

- [ ] Added required new tests relevant to the changes
- [ ] Updated Documentation as relevant to the changes
- [ ] PR change contains code related to security
- [ ] PR introduces changes that break compatibility with other modules/services (If YES, please provide description)
- [ ] Run `go fmt` or `format-python.sh` as applicable
- [x] Update Changelog
- [x] Integration tests are passing
- [ ] If Cloudadapter changes, check Azure connectivity manually

# _Code must act as a teacher for future developers_

### CODE REVIEW

File: inbm/dispatcher-agent/fpm-template/etc/apparmor.d/usr.bin.inbm-dispatcher [hunk 1]
```diff
@@ -47,7 +47,7 @@
   /etc/dpkg/dpkg.cfg r,
   /etc/dpkg/dpkg.cfg.d/ r,
   /etc/dpkg/dpkg.cfg.d/* r,
-  /etc/intel_edge_node/tokens/platform-update-agent/rs_access_token r,
+  /etc/intel_edge_node/tokens/release-service/rs_access_token r,
```
Comment (gblewis1) [inline inbm/dispatcher-agent/fpm-template/etc/apparmor.d/usr.bin.inbm-dispatcher]: according to Matenko, Tadeusz: "The correct path is /etc/intel_edge_node/tokens/release-service/access_token"

File: inbm/dispatcher-agent/dispatcher/sota/constants.py [hunk 2]
```diff
@@ -18,7 +18,7 @@
 TIBER_UPDATE_TOOL_PATH = get_canonical_representation_of_path('/usr/bin/os-update-tool.sh')
 
 # Release server access token path
-RELEASE_SERVER_TOKEN_PATH = get_canonical_representation_of_path('/etc/intel_edge_node/tokens/platform-update-agent/'
+RELEASE_SERVER_TOKEN_PATH = get_canonical_representation_of_path('/etc/intel_edge_node/tokens/release-service/'
```
Comment (gblewis1) [inline inbm/dispatcher-agent/dispatcher/sota/constants.py:21]: according to Matenko, Tadeusz: "The correct path is /etc/intel_edge_node/tokens/release-service/access_token"

File: inbm/dispatcher-agent/fpm-template/etc/apparmor.d/usr.bin.inbm-dispatcher (hunk 1 above)
Comment (yengliong93) [reply inbm/dispatcher-agent/fpm-template/etc/apparmor.d/usr.bin.inbm-dispatcher]: Path changed to /etc/intel_edge_node/tokens/release-service/access_token

File: inbm/dispatcher-agent/dispatcher/sota/constants.py (hunk 2 above)
Comment (yengliong93) [reply inbm/dispatcher-agent/dispatcher/sota/constants.py:21]: Path changed to /etc/intel_edge_node/tokens/release-service/access_token

**Review by gblewis1 — APPROVED** [review]

**Pull request merged**